src/openfermion/hamiltonians/_plane_wave_hamiltonian_test.py

```diff
@@ -126,3 +126,20 @@ def test_plane_wave_energy_cutoff(self):
 
         max_diff = numpy.amax(numpy.absolute(spectrum_1 - spectrum_2))
         self.assertGreater(max_diff, 0.)
+
+    def test_plane_wave_period_cutoff(self):
+        geometry = [('H', (0,)), ('H', (0.8,))]
+        grid = Grid(dimensions=1, scale=1.1, length=5)
+        period_cutoff = 50.0
+
+        h_1 = plane_wave_hamiltonian(grid, geometry, True, True, False, None)
+        jw_1 = jordan_wigner(h_1)
+        spectrum_1 = eigenspectrum(jw_1)
+
+        h_2 = plane_wave_hamiltonian(grid, geometry, True, True, False, None,
+                                     True, period_cutoff)
+        jw_2 = jordan_wigner(h_2)
+        spectrum_2 = eigenspectrum(jw_2)
+
```

Comment:
Same comment for ths test, I think it only really makes sense to test against the potential operators and not have the kinetic included, but if this is only to placehold until integration testing, it might be okay.

Reply:
Yes, this test is really only for code coverage, and a placeholder for future integration test. Added a TODO. 

Also, as mentioned in our email thread, since plane wave external potential is implemented as an inverse Fourier transformation of dual basis external potential:
https://github.com/quantumlib/OpenFermion/blob/master/src/openfermion/hamiltonians/_plane_wave_hamiltonian.py#L97 ,
We are not really able to do integration test for external potential.